README.md

```diff
@@ -185,12 +189,14 @@ logger:
 
 or use this service call without restart HA:
 
-> `Developer Tools` -> `Services` -> `GO TO YAML MODE`
-> paste below yaml content to the form，and run `CALL SERVICE`
+> `Developer Tools` -> `Actions` -> select `Logger: Set Log Level` -> `GO TO YAML MODE`
+> paste below yaml content to the form，and run
 
 ```yaml
-service: logger.set_level
+action: logger.set_level
 data:
   custom_components.midea_ac_lan: debug
   midealocal: debug
 ```
+
+> Tips: this mode not required reboot, but it can't capture device startup error log, recommend to edit `configuration.yaml` file to enable debug mode.
```

Comment:
```suggestion
> Tips: this mode doesn't require reboot, but it can't capture device startup error log, so is still recommended to edit `configuration.yaml` file to enable debug mode.
```